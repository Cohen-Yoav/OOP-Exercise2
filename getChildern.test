# getChildren method Testing file

CreateGraph graph1
CreateNode n1 5
AddNode graph1 n1
ListChildren graph1 n1

CreateNode n2 2
ListChildren graph1 n2

AddNode graph1 n2
AddEdge graph1 n1 n2
ListChildren graph1 n1

CreateNode n3 9
AddNode graph1 n3
AddEdge graph1 n1 n3
ListChildren graph1 n1

CreateNode n4 4
CreateNode n5 5
CreateNode n6 5
AddNode graph1 n4
AddNode graph1 n5
AddNode graph1 n6
AddEdge graph1 n1 n4
AddEdge graph1 n1 n5
AddEdge graph1 n1 n6
ListChildren graph1 n1
