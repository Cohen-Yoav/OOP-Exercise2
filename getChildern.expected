# getChildren method Testing file

created graph graph1
created node n1 with cost 5
added node n1 to graph1
the children of n1 in graph1 are:

created node n2 with cost 2

added node n2 to graph1
added edge from n1 to n2 in graph1
the children of n1 in graph1 are: n2

created node n3 with cost 9
added node n3 to graph1
added edge from n1 to n3 in graph1
the children of n1 in graph1 are: n2 n3

created node n4 with cost 4
created node n5 with cost 5
created node n6 with cost 5
added node n4 to graph1
added node n5 to graph1
added node n6 to graph1
added edge from n1 to n4 in graph1
added edge from n1 to n5 in graph1
added edge from n1 to n6 in graph1
the children of n1 in graph1 are: n2 n3 n4 n5 n6
